themes/src/main/resources/theme/keycloak.v2/login/register.ftl

```diff
@@ -76,9 +74,13 @@
 
             document.getElementById("password").addEventListener("change", (event) => {
                 const serverErrors = document.getElementById("input-error-password");
-                if (serverErrors) {
-                    serverErrors.remove();
-                }
+                if (serverErrors) serverErrors.remove();
```

Comment:
Let's not make any code style changes, please add brackets to `if` statements.

Reply:
You're right. Reverted this change.